scripts/upload-images

```diff
@@ -84,6 +84,13 @@ for image in "executionrole" "busybox" "nginx" "python" "ubuntu" "consul" "debia
   fi
 done
 
+REGISTRY_IMAGE_CONTAINER="test-ecs-registry-container:arm"
+REGISTRY_IMAGE="https://github.com/tianon/docker-registry-image.git#multiarch:2.6"
```

Comment:
is there an official repo for this?

Reply:
Yes they added support for Arm 😄  Updating.